packages/api/src/utils/cronJobs/TVL.ts

```diff
@@ -1,20 +1,16 @@
 import { Predicate, TotalValueLocked } from '@src/models';
-import { IConfVault, Vault } from 'bakosafe';
-import { BN, bn } from 'fuels';
+import { Asset, IConfVault, Vault } from 'bakosafe';
 import cron from 'node-cron';
-import axios from 'axios';
-
-const assetId =
-  '0xf8f8b6283d7fa5b672b530cbb84fcccb4ff8dc40f8176ef4544ddb1f1952ad07';
+import { assetsMapById } from '../assets';
+import app from '@src/server/app';
 
 // Executa todos os dias a meia-noite
 const TVLCronJob = cron.schedule('0 0 * * *', async () => {
   console.log('[TVL CRON JOB] running');
   try {
-    //Validação para limitar apenas um registro diário por assetId
+    //Validação para limitar apenas um registro diário
     const savedTVL = await TotalValueLocked.createQueryBuilder('tvl')
-      .where('tvl.assetId = :assetId', { assetId })
-      .andWhere('DATE(tvl.created_at) = CURRENT_DATE')
+      .where('DATE(tvl.created_at) = CURRENT_DATE')
```

Comment:
would the validation be just the exact date?
try:
`  .where('tvl.created_at >= CURRENT_DATE AND tvl.created_at < CURRENT_DATE + INTERVAL '1 DAY'')
`

Reply:
Only checking by the current date seems to satisfy the condition of having only one record per day. It checks whether there is any record in the table created on the current date and bases the result on that:
     - If a record for the current day already exists, the cron job should stop.
     - If it doesn't, the cron job should continue and create the necessary records.